react/BarContextProvider/indes.spec.jsx

```diff
@@ -0,0 +1,64 @@
+import React from 'react'
+
+import CozyClient, {
+  createMockClient,
+  CozyProvider,
+  useClient
+} from 'cozy-client'
+import configureStore from 'redux-mock-store'
+import { Provider, useStore } from 'react-redux'
+
+import BarContextProvider from '.'
+import I18n, { useI18n, translate } from '../I18n'
+const locales = { helloworld: 'Hello World !' }
+const localesBar = { a: 'b' }
+const DumbHelloWorld = translate()(({ t, f, lang }) => (
```

Comment:
Nit: Please add a blank line between component declarations and the surrounding code.